src/Tests/dotnet-MsiInstallation.Tests/WorkloadSetTests.cs

```diff
@@ -213,9 +213,43 @@ public void UpdateToWorkloadSetVersionWithManifestsNotAvailable()
             GetWorkloadVersion().Should().Be(workloadVersionBeforeUpdate);
         }
 
-        string GetWorkloadVersion()
+        [Fact]
+        public void UpdateWorkloadSetViaGlobalJson()
+        {
+            InstallSdk();
+
+            var versionToUpdateTo = "8.0.300-preview.0.24217.2";
+            var directory = "C:\\SdkTesting";
+
+            string originalVersion = GetWorkloadVersion();
+
+            var rollback = GetRollback(directory);
+
+            VM.WriteFile("C:\\SdkTesting\\global.json", @"{""sdk"":{""workloadVersion"":""versionToUpdateTo""}}".Replace("versionToUpdateTo", versionToUpdateTo)).Execute().Should().Pass();
```

Comment:
nit: is there a way to use the variable inline rather than the clunky replace logic? Just a bit wordy.

Reply:
String interpolation is a bit hampered when the string you're creating actually needs { and }. I can try to see if there's an escape sequence for that, though.